docs/hub-tutorials/join-testnet.md

```diff
@@ -4,64 +4,159 @@ order: 4
 
 # Join the Public Testnet
 
-::: tip Current Testnet
-See the [testnet repo](https://github.com/cosmos/testnets) for
-information on the latest testnet, including the correct version
-of Gaia to use and details about the genesis file.
-:::
+| Current Upgrade | Chain Id       | Upgrade Block Height | Upgrade Date     |
+| --------------- | -------------- | -------------------- | ---------------- |
+| Vega            | `vega-testnet` | `7368587`            | November 12 2021 |
+
+
+## Background
+The Cosmos Hub Testnet is currently running after it's most recent [Vega Upgrade](https://interchain-io.medium.com/cosmos-hub-vega-upgrade-testnet-details-e9c5d69a59c). Visit the [testnet explorer](https://vega-explorer.hypha.coop/) to view all on chain activity.
+
+For those who just need instructions on performing the upgrade, see the [Upgrade](#upgrading) section.
+
+## Prerequisites
+
+**Hardware**
+
+It's recommended that public testnet nodes are running on machines with at least `16GB` of RAM.
+
+**Make sure Go & Gaia are [properly installed](../getting-started/installation.md). The most recent Gaia version for the Vega Testnet is [`v6.0.0-rc3`](https://github.com/cosmos/gaia/tree/v6.0.0-rc3)**
+
+
+This tutorial will provide all necessary instructions for joining the current public testnet. Getting familiar with the installation and startup process for running a mainnet node can provide additional understanding and context. See [Join Mainnet](./join-mainnet.md) for a detailed walkthrough, or the [Quickstart Guide](../getting-started/quickstart.md) for streamlined instructions.
```

Comment:
We're pointing them in the direction of the mainnet tutorial in case they want additional information on configuring a node...this doesn't seem like the place to duplicate that info

Reply:
Yeah, i'm onboard with that. But doubt Quickstart is necessary as the statesync instructions are pretty much identical and quicksync isn't an option for testnets. Maybe this rewrite is more clear? Wdyt?

```suggestion
This tutorial will provide all necessary instructions for joining the current public testnet. If you're interested in more advanced configuration and synchronisation options, see [Join Mainnet](./join-mainnet.md) for a detailed walkthrough.
```

Reply:
Yeah that sounds great